.github/workflows/linter.yml

```diff
@@ -0,0 +1,16 @@
+name: Code checking
```

Comment:
nit; rename `Code checking` to `Lint Code` or something like that.

Reply:
Did you forget to commit your change?

Reply:
Seems like the beginner mistake : forgot to push.